### PR TITLE
KTOR-1114 Add performance monitoring to all core entries

### PR DESCRIPTION
**Pull request by hhariri**

### CODE REVIEW

File: .teamcity/src/subprojects/build/core/NativeBuild.kt [hunk 1]
```diff
@@ -18,6 +19,10 @@ class NativeBuild(private val osEntry: OSEntry) : BuildType({
             jdkHome = "%env.JDK_11%"
         }
     }
+    features {
```
Comment (Stexxe) [inline .teamcity/src/subprojects/build/core/NativeBuild.kt:22]: Code duplication

File: .teamcity/src/subprojects/build/core/JavaScriptBuild.kt [hunk 2]
```diff
@@ -23,10 +24,15 @@ class JavaScriptBuild(private val javaScriptEngine: JavaScriptEngine) : BuildTyp
...
     }
+    features {
+        perfmon {
+        }
+    }
     requirements {
         contains("teamcity.agent.jvm.os.name", "Linux")
         noLessThan("teamcity.agent.hardware.memorySizeMb", "7000")
     }
+
```
Comment (Stexxe) [inline .teamcity/src/subprojects/build/core/JavaScriptBuild.kt:35]: Unnecessary change

File: .teamcity/src/subprojects/build/core/JavaScriptBuild.kt [hunk 3]
```diff
@@ -23,10 +24,15 @@ class JavaScriptBuild(private val javaScriptEngine: JavaScriptEngine) : BuildTyp
             setupDockerForJavaScriptTests(javaScriptEngine)
         }
     }
+    features {
+        perfmon {
```
Comment (Stexxe) [inline .teamcity/src/subprojects/build/core/JavaScriptBuild.kt:28]: Maybe better to have open and closed curly brackets on the same line if a block is empty: `perfmon {}`

**Review by Stexxe — CHANGES_REQUESTED** [review]

File: .teamcity/src/subprojects/build/core/NativeBuild.kt (hunk 1 above)
Comment (hhariri) [reply .teamcity/src/subprojects/build/core/NativeBuild.kt:22]: Yes. Each block has this and it's required and if it diverges will add contents there
